### DISCUSSION TITLE
What happened to includedRoutes ?

### DISCUSSION THREAD

**Discussion by KarlBaumann** [gatsbyjs/gatsby#30376] [Help] [answered]
`warn Warning: there are unknown plugin options for "gatsby-source-wordpress": includedRoutes
Please open an issue at ghub.io/gatsby-source-wordpress if you believe this option is valid.`

How do you define this in the latest version?

**Comment by jasonbahl** [comment, marked as answer]
@KarlBaumann If I'm not mistaken this option was a way to specify REST API routes to include in Gatsby. 

`gatsby-source-wordpress` v4+ no longer uses the WP REST API but instead uses WPGraphQL. 

Since WPGraphQL is used, there are no longer REST routes to include or exclude. Instead, there's GraphQL Types. 

While not exact same as the `includedRoutes`, the new source plugin allows fine-grain control of each Type in the WPGraphQL Schema, allowing for fine-grain control over how many objects of each type are fetched, whether they're included or excluded, etc: https://github.com/gatsbyjs/gatsby/blob/master/packages/gatsby-source-wordpress/docs/plugin-options.md#type
